src/schema/project-checkbook.ts

```diff
@@ -0,0 +1,21 @@
+import { char, numeric, pgTable, text, uuid } from "drizzle-orm/pg-core";
+import { z } from "zod";
+import { managingCodeEntitySchema } from "./managing-code";
+
+export const projectCategory = pgTable("project_category", {
+  id: uuid("id"),
+  managingCode: char("managing_code", { length: 3 }),
```

Comment:
It's a buried in the [sql table definition](https://github.com/NYCPlanning/ae-data-flow/blob/f513c4a44751497b48a6926a3047da2b1ead9ec9/sql/commitment.sql#L148) and not obvious in the [drizzle foreign key docs](https://orm.drizzle.team/docs/indexes-constraints#foreign-key). However, we'll also want to add a multicolumn foreign key.